programs/unrolled.py

```diff
@@ -0,0 +1,913 @@
+# Single QUA script generated at 2024-02-07 17:07:09.341746
+# QUA library version: 1.1.6
+
+# Logged output for this program:
+# Connection time: 1.482797622680664
+# Execution time: 3.4133341312408447
+# readout(2000, 0.005, Rectangular())_0_I (30,)
+# readout(2000, 0.005, Rectangular())_0_Q (30,)
+# Fetch time: 39.18629169464111
+# Total time: 44.08242344856262
+
+
+from qm.qua import *
+
+with program() as prog:
+    v1 = declare(
+        int,
+    )
+    v2 = declare(
+        fixed,
+    )
+    v3 = declare(
+        fixed,
+    )
+    wait((4 + (0 * (Cast.to_int(v2) + Cast.to_int(v3)))), "readout0")
+    with for_(v1, 0, (v1 < 4096), (v1 + 1)):
+        align()
+        play("drive(40, 0.0025, Gaussian(5))", "drive0")
+        frame_rotation_2pi(-0.013333333333333334, "drive0")
+        play("drive(40, 0.0025, Gaussian(5))", "drive0")
+        reset_frame("drive0")
+        wait(21, "readout0")
+        measure(
+            "readout(2000, 0.005, Rectangular())",
+            "readout0",
+            None,
+            dual_demod.full("cos", "out1", "sin", "out2", v2),
+            dual_demod.full("minus_sin", "out1", "cos", "out2", v3),
+        )
+        r1 = declare_stream()
+        save(v2, r1)
+        r2 = declare_stream()
+        save(v3, r2)
+        wait(75501, "drive0")
```

Comment:
These `wait` instructions have no direct correspondent in the for loop, I guess they are some kind of padded(/batched?) replacement of the initial `align()`.

Why not using an `align()` also here?